utils/clone-llvm.sh

```diff
@@ -14,7 +14,7 @@
 #
 ##===----------------------------------------------------------------------===##
 
-export commithash=6577f95f9fb04fb7def7a5ae78bcf5f2b4f21b6e
```

Comment:
FWIW - you could replace all of the clones and build with 

* `pip download mlir -f https://github.com/Xilinx/mlir-aie/releases/expanded_assets/mlir-distro`
* `pip download mlir-aie -f https://github.com/Xilinx/mlir-aie/releases/expanded_assets/latest-wheels` 

You can pin with e.g. `==18.0.0.2023120309+9584f583` too.

Reply:
Note, this won't work if you decide you want to build without RTTI here.

Reply:
> Note, this won't work if you decide you want to build without RTTI here.

I would love to use the wheels as part of the build flow instead of this. In the short term having RTTI off is a requirement though...

Reply:
i can turn build `no-rtti` wheels for you guys.

Reply:
That would be super cool. Once the no-rtti wheels are ready I'll create a new PR to replace this bit.